src/main/kotlin/racing/CarRacingResultView.kt

```diff
@@ -1,21 +1,26 @@
 package racing
 
-class CarRacingResultView {
+object CarRacingResultView {
     fun printCarRacingResult(cars: List<Car>, moveCount: Int) {
         // turnOfPosition의 0번 index는 기본으로 1이 셋팅되어있기때문에
         // 범위를 0 <= moveIndex <= moveCount 로 설정
         for (moveIndex in 0..moveCount) {
-            cars.forEach { car ->
-                printCurrentCarMovingDistance(car.turnOfPosition[moveIndex])
-            }
-            println()
+            printMovingDistanceByTurn(cars = cars, moveIndex = moveIndex)
```

Comment:
```suggestion
            printMovingDistanceByTurn(cars = cars) { car -> car.turnOfPosition[moveIndex] }
```
위 코드를 통해 개선해 보면 어떨까요?